changelog.d/gh-6071.changed

```diff
@@ -0,0 +1,6 @@
+generic mode: allow text input without human-readable indentation up to 500
```

Comment:
Can't we handle more than 500 bytes without problems? If we don't raise it then I think we should make it configurable.

Reply:
No, minified or binary code can be extremely slow to handle, so we skip such files. Allowing single-line files up to 500 bytes is a quick fix to prevent a common problem that rule authors run into. The generic mode is for source code, i.e. human-readable text with meaningful indentation.

Removing this and other limits completely can be done with the `--force` option of `spacegrep` to developers who have access to it.